tests/Phpixela/ApiTest.php

```diff
@@ -0,0 +1,245 @@
+<?php
+
+namespace StandardTest;
+
+use function GuzzleHttp\Psr7\try_fopen;
+use PHPUnit\Framework\TestCase;
+use Phpixela\Client;
+
+/**
+ * Class StandardTest
+ * @package CollectionTest
+ */
+class ApiTest extends TestCase
+{
+    /** @var $user_name string */
+    const user_name = "phpixela1";
```

Comment:
I think we should use mock instead of real API.

Reply:
True. 
Unnecessary to test the test. 
So I will delete it.